<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Penalty</class>
 <widget class="QDialog" name="Penalty">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1359</width>
    <height>921</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Penalty</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="teamLeftButton">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Team Left</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="teamRightButton">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Team Right</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QFrame" name="frame">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QRadioButton" name="blueButton">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Blue Card</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="yellowButton">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Yellow Card</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="yellowredButton">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Yellow, Red Card</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="redButton">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Red Card</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Player number:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="input_number">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>OR choose Player:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="list_players"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Give reason:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="input_reason">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>OR choose one:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="list_reasons"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>Penalty</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>596</x>
     <y>875</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>796</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>Penalty</receiver>
   <slot>ok()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>761</x>
     <y>879</y>
    </hint>
    <hint type="destinationlabel">
     <x>903</x>
     <y>763</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>teamLeftButton</sender>
   <signal>clicked()</signal>
   <receiver>Penalty</receiver>
   <slot>team_chosen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>455</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>teamRightButton</sender>
   <signal>clicked()</signal>
   <receiver>Penalty</receiver>
   <slot>team_chosen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>775</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>437</x>
     <y>436</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ok()</slot>
  <slot>team_chosen()</slot>
 </slots>
</ui>
